README.md

```diff
@@ -1,2 +1,27 @@
 # Python User Language Support
-Python User Language Support for [Cloudstate](https://github.com/cloudstateio/cloudstate).
\ No newline at end of file
+Python User Language Support for [Cloudstate](https://github.com/cloudstateio/cloudstate).
+
+## Installation via source
+
+```
+> git clone https://github.com/cloudstateio/python-support.git
+Cloning into 'python-support'...
+
+> cd python-support
+> python3 -m venv ./venv 
+> source ./venv/bin/activate
+> python --version     
+Python 3.7.3
+> pip --version 
+> pip install -r requirements.txt
+```
+
+### generate installer
+```
+./venv/bin/python3 setup.py bdist_wheel
```

Comment:
@sleipnir 
I get this executing bdist_wheel. I read I need ti `pip install wheel`?
```
(venv) ➜  python-support git:(master) ✗ ./venv/bin/python3 setup.py bdist_wheel
package name: cloudstate, version: 0.5.0
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'bdist_wheel'
```

Reply:
Yeah, after
```
pip install wheel
```

the next command installed the module.

Reply:
Yeah! That was missing to mention, but we need it if we want to publish